src/components/Item/TextItem/Item.js

```diff
@@ -60,10 +61,11 @@ const TextItem = props => {
                 <Line />
                 <div className="dashboard-item-content">
                     <RichTextEditor onEdit={onChangeText}>
-                        <TextField
+                        <Input
                             value={text}
                             multiline
                             rows={1}
+                            rowsMax={15}
```

Comment:
Any reason for adding this? I had to remove it for fixing another bug, see: https://github.com/dhis2/dashboards-app/commit/9419d281f477ce61b75352f1d1ef7ef1c152b0a3.

Reply:
Right - I knew there was something. Fixed it by removing both the `rows` and `rowsMax` properties.